tests/kafkatest/tests/client/test_producer_consumer_compat.py

```diff
@@ -53,8 +53,10 @@ def min_cluster_size(self):
         # Override this since we're adding services outside of the constructor
         return super(TestProducerConsumerCompat, self).min_cluster_size() + self.num_producers + self.num_consumers
 
-    # TODO: when KAFKA-4462 is fully implemented, we should test other versions here.
     @parametrize(broker_version=str(TRUNK))
+    @parametrize(broker_version=str(V_0_10_0_0))
```

Comment:
OK, let's just test those.  We shouldn't add major features in point releases anyway.